SasquatchMac/SasquatchMac/ViewControllers/AppCenterViewController.swift

```diff
@@ -48,7 +48,7 @@ class AppCenterViewController : NSViewController, NSTextFieldDelegate, NSTextVie
   override func viewDidLoad() {
     super.viewDidLoad()
     installIdLabel?.stringValue = appCenter.installId()
-    appSecretLabel?.stringValue = prodAppSecret()
+    appSecretLabel?.stringValue = appCenter.appSecret()
```

Comment:
I think showing app secret as empty will make sense more than showing the app secret when app startups with one collector mode. So I use the `prodAppSecret` method.